doc/user/content/sql/commit.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "COMMIT"
+menu:
+  main:
+    parent: "sql"
+---
+
+`COMMIT` commits the current transaction.
+
+## Syntax
+
+{{< diagram "commit.svg" >}}
+
+## Details
+
+If the current transaction is **write only**, the changes made by the transaction become visible to others.
```

Comment:
I think rather than stating something about the visibility, making mention that writes occur at the transaction's chosen time? This makes it nominally more clear that if someone else has a transaction open, they will not be able to see these values because their timestamp will be in the past.

Reply:
done